docker/mproxy.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Mainflux
+# SPDX-License-Identifier: Apache-2.0
+
+version: "3.7"
+
+volumes:
+  mainflux-mqtt-redis-volume:
+
+services:
+  nginx:
+    depends_on:
+      - mqtt-adapter
+
+  emqx:
+    image: emqx/emqx:latest
```

Comment:
Do we use Verne and drop EMQX?

Reply:
docker-compose stuff is not included in this PR. It's already huge. I will send separate PR

> Besides these remarks, in general looks like there is a huge number of vendor files pulled - this is suspicious. Please re-check.

Yea vendoring is huge, not sure actually how to check its your code :) 
What I did I just update with `go mod tidy` and `go mod vendor` to sync. Let me try again.

Reply:
docker-compose stuff is not included in this PR. It's already huge. I will send separate PR